generate/template/astronomy.py

```diff
@@ -374,7 +374,7 @@ def _UserDefinedStar(body):
         return star
     return None
 
-def DefineStar(body, ra, dec, distanceLightYears):
+def DefineStar(body: Body, ra: float, dec: float, distanceLightYears: float):
```

Comment:
Is there any advantage to marking the return type with `-> None` here, like you did with some other functions that don't return anything? I don't know enough about Python type hints.

Reply:
Yes, if you don't explicitly define the return type as `None`, [it defaults to `Any`](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#the-any-type) which is pretty much the same as not type hinting at all.

For constructors, [PEP 484](https://peps.python.org/pep-0484/) recommends hinting `None`.